Submissions/mohdanss.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Muhammad Ans",
+  "img": "https://avatars.githubusercontent.com/u/58670177?v=4",
+  "email": "mohd.ans.riaz@gmail.com",
+  "links": {
+    "website": "mohdanss.github.io/portfolio",
```

Comment:
please provide the full URL including http or https

Reply:
Updated the website url with 'https://'